charts/turbinia/values-production.yaml

```diff
@@ -0,0 +1,381 @@
+## Turbinia Helm Production Values
```

Comment:
Maybe add a comment about how the copy in the root dir of this project needs to be updated if this is updated?

Reply:
Going to add this as an AI for future maintenance instructions as some more housekeeping items to add